README.md

```diff
@@ -144,3 +147,329 @@ Otherwise you will need to tell the VM each time as well like so:
+
+```
+##fileFormat  = tsv
+##commandLine = org.broadinstitute.hellbender.tools.exome.ExomeReadCounts  ...snip...
+##title       = Read counts per target and sample
+CONTIG  START   END     NAME    SAMPLE1
+1       12200   12275   target1    1.150e-05
+1       13505   13600   target2    1.500e-05
+1       31000   31500   target3    7.000e-05
+....snip....
+```
+
+###### Invocation
+
+```
+ java -Xmx8g -jar <path_to_hellbender_jar> ExomeReadCounts -I <input_bam_file> -O <pcov_output_file_path>  -exome <target_BED> -R <ref_genome> \ 
+       -transform PCOV -exonInfo FULL -groupBy SAMPLE -keepdups
+```
+
+##### Step 2. Create coverage profile
+
+###### Inputs
+- proportional coverage file from Step 1
+- target BED file -- must be the same that was used for the PoN
+- PoN file
+- directory containing the HDF5 JNI native libraries
+
+###### Outputs
+- target file (tsv) -- details each target with chromosome, start, end, and log copy ratio estimate
+```
+#fileFormat = tsv
+#commandLine = ....snip....
+#title = ....snip....
+name    contig  start   stop    SAMPLE1
+target1    1       12200   12275   -0.5958351605220968
+target2    1       13505   13600   -0.2855054918109098
+target3    1       31000   31500   -0.11450116047248263
+....snip....
+```
+- pre-tangent-normalization target file (tsv) -- same as target file (tsv) above, but copy ratio estimates are before the noise reduction step.  The file format is the same as the target file (tsv).
+- fnt file (tsv) -- proportional coverage divided by the target factors contained in the PoN.  The file format is the same as the proportional coverage in step 1.
+- betaHats (tsv) -- used by developers and evaluators, typically, but output location must be specified.
+
+###### Invocation
+```
+java -Djava.library.path=<hdf_jni_native_dir> -Xmx8g -jar <path_to_hellbender_jar> NormalizeSomaticReadCounts -I <pcov_input_file_path> -T <target_BED> -pon <pon_file> \
+ -O <output_target_cr_file> -FNO <output_target_fnt_file> -BHO <output_beta_hats_file> -PTNO <output_pre_tangent_normalization_cr_file>
+```
+
+
+##### Step 3. Segment coverage profile
+
+###### Inputs
+- target file (tsv) -- from step 2.
+- sample name
+
+###### Outputs
+- seg file (tsv) -- segment file (tsv) detailing contig, start, end, and copy ratio (segment_mean) for each detected segment.  Note that this is a different format than python recapseg, since the segment mean no longer has log2 applied.
+```
+Sample  Chromosome      Start   End     Num_Probes      Segment_Mean
+SAMPLE1        1       12200   70000   18       0.841235
+SAMPLE1        1       300600  1630000 337     1.23232323
+....snip....
+```
+
+###### Invocation
+
+```
+java -Xmx8g -jar <path_to_hellbender_jar>  PerformCBSSegmentation  -S <sample_name> -T <target_file_tsv> -O <output_seg_file> -log
```

Comment:
Could we rename this tool to just `PerformSegmentation` without the CBS?  Why commit to a specific implementation, and why expose users to implementation details?